index.html

```diff
@@ -0,0 +1,257 @@
+      <h2 class="main-program-header light-grey heading-medium">
+        Main Program
+      </h2>
+      <hr class="horizontal-line text-center" />
+      <div class="program-items">
+        <article class="program-item flex-row-center bg-light-black">
+          <div class="item item-icon heading-big">
+            <i class="fas fa-book-reader light-grey"></i>
+          </div>
+          <h3 class="item item-header dark-orange">Lecture</h3>
+          <p class="item item-content light-grey">
+            Asian Youth Program (AYP), in partnership with Womankind (formerly New York Asian Women’s Center)
+          </p>
+        </article>
+        <article class="program-item flex-row-center bg-light-black">
+          <div class="item item-icon heading-big">
+            <i class="fab fa-accusoft light-grey"></i>
+          </div>
+          <h3 class="item item-header dark-orange">Exhibition</h3>
+          <p class="item item-content light-grey">
+            Asian Youth Program (AYP), in partnership with Womankind (formerly New York Asian Women’s Center)
+          </p>
+        </article>
+        <article class="program-item flex-row-center bg-light-black">
+          <div class="item item-icon heading-big">
+            <i class="far fa-comments light-grey"></i>
+          </div>
+          <h3 class="item item-header dark-orange">Forum</h3>
+          <p class="item item-content light-grey">
+            Asian Youth Program (AYP), in partnership with Womankind (formerly New York Asian Women’s Center)
+          </p>
+        </article>
+        <article class="program-item flex-row-center bg-light-black">
+          <div class="item item-icon heading-big">
+            <i class="fas fa-code-branch light-grey"></i>
+          </div>
+          <h3 class="item item-header dark-orange">Workshop</h3>
+          <p class="item item-content light-grey">
+            Asian Youth Program (AYP), in partnership with Womankind (formerly New York Asian Women’s Center)
+          </p>
+        </article>
+        <article class="program-item flex-row-center bg-light-black">
+          <div class="item item-icon heading-big">
+            <i class="fas fa-chart-line light-grey"></i>
+          </div>
+          <h3 class="item item-header dark-orange">Ignite</h3>
+          <p class="item item-content light-grey">
+            Asian Youth Program (AYP), in partnership with Womankind (formerly New York Asian Women’s Center)
+          </p>
+        </article>
+      </div>
+      <button type="button" name="join-us" class="btn main-btn">
+        Join the Program
+      </button>
+      <a class="see-program"><u>SEE THE WHOLE PROGRAM</u></a>
+    </section>
+    <section class="featured-speakers flex-column-center" id="speakers">
+      <h2 class="featured-speakers-header light-black heading-medium">
+        Featured Speakers
+      </h2>
+      <hr class="horizontal-line text-center" />
+      <div class="speakers-list"></div>
+      <button
```

Comment:
- [ ] OPTIONAL: The featured speakers section was aligned in two columns when I first reviewed, now it looks like the changes you made affected this section and it became a single column. I strongly recommend to bring back the previous implementation 👍🏻 

Your design  | Behance design
------------- | -------------
<img width="710" alt="Screenshot 2022-06-30 at 08 45 21" src="https://user-images.githubusercontent.com/77038610/176594703-e53d4ec8-7c9d-4e97-9c17-d03d04804ee1.png"> | <img width="866" alt="Screenshot 2022-06-30 at 08 47 03" src="https://user-images.githubusercontent.com/77038610/176594862-3400eb37-e099-4d41-a7c4-3177ed60d2c8.png">